scripts/sysman/memoryBwCounterValidBits.yml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Memory Bandwidth Counter Valid Bits"
+version: "1.8"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Memory Bandwidth Counter Valid Bits Extension Name"
+version: "1.8"
+name: $S_MEMORY_BANDWIDTH_COUNTER_VALID_BITS_EXP_NAME
+value: '"$XS_extension_mem_bandwidth_counter_valid_bits"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Memory Bandwidth Counter Valid Bits Extension Version(s)"
+version: "1.8"
+name: $s_mem_bandwidth_counter_bits_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties for reporting valid bit count for memory bandwidth counter value"
+version: "1.8"
+class: $sMemory
+name: $s_mem_bandwidth_counter_bits_exp_t
```

Comment:
@AshwinKumarKulkarni  agree it should be properties_t since you are deriving from base_properties. So, could you also please rename the extension name in line 16, 17 above?

Reply:
Added properties in the name and at line 16,17.